community/aqua/general/amplitude_estimation.ipynb

```diff
@@ -0,0 +1,231 @@
sry+BIy7yQFPJ0kxRIAy8ICByHWcMB0HM1XxiGEAQBBw4cwIEDB6DrOu7dvYsbN2/C9zzMLyzg5MmTzC0TkGXxm2+8AUEQcO7cudTKu7YYQGk8hUiaR1u2HWXIfIbyhoMg6D0tcBu4eu0a9szODqywi1q3WSb0VCR86JZl4dM7d3D37l1Yto3Z2Vm88MILLP2WZVb1en+j9blKu0csy4LrushpWtvYAxeT0bAtazO2w3HwXBeSLGOENl3/HGNI7ASSJIGX5ejGyeVYILVQKLRkjfRTXs1cG8Qa6ZXW0rJq8vk8jh0/jmPHj2NpaQnXrl7Faz/6EcYqFezfvx+e4+Ddd97B9PQ0nnj88Z6bMDONkAFpcauaBrfRgE3IfTt40FkxWa7dICx213Fw+/ZtfGVA1rplWfD7zVkn4AB4vo+79+5h7t49bKyvY2ZmBqcefzwqrksQMFvt9WAYhLHt6P9JuJ4H07Igy3L7BjKxDDHf9yO3S6HAJhrXdVEZGxtsNfQjiiGxE3Ach1yxiObaGhTfh6aq0A0jsjJjNwITterhQeZ4PrLUidg/3UdadWsS7QS84piamkKxWMTyygos08TS4iIMw4CiaZBlGesbGxgZGenZvx1PadsuJEmCKAisyXa/VjtLrRvgiqITgiDo2U/cL65dv46ZmRlomsbSLPuF53kwTBOSKPZMZEEYolatYmVlBYuLi1heXcX42Bj279uH5557rqtbqhdyT5J6GvwggKHr4DsEf5P7sCwLHMCMiCAIwIkiJiYnH5iu/sPEkNhjKBQKMJpNWI6DQi4HyXFgEf9kCxERcsn62PExhTwGUpUa+H7XfN4sx8nlchgdGUGV4zA1PY16vY79Bw/CME28/957aBoGxsfGMDU1hcnJSZSKxUxByEHk8lNomoZ6oxHp4G/TauJJGmonBGSVRC02ei2Zq4BsTzs+sQbQ0T9AGML3fVYIdj/heR5u3rqFl86d2/a+giBgWTBZMpHCMIRhGFhaXsbK8jKWlpehqComx8exb98+HD12DJVyOeoVmnEMWQ2gtA5N8bsyCAI0m02EAIr5fDopt3HbcLFWf47jYGLXrpYK1M8zhsQeA8/zKJRKqK+vw/M85EhKoaHryMc71/RoLVL53CAIkLR1OrXqytoUg6JUKsHzPNTrdXCiiGKxiEOHDgGnTkUiYisrWF5exvXr1+F5HiP5iYmJ9gRAyG4QLhlJliGKIizTjOIX2/C1e0EA3/MQkrzkkFzfwPej30Cm7yhE5IeOPr4ZsAsBhJ6Hpq6zHHCe46IVF89DIMJYPM+zZtHbwc2bNzExPj6QDkm6YXQNMNL7YYkQeej7mJiYwNT0NB5//HHmk7dtm6kmBmFv0r0cAHSRIEh7L65M2mg2EYYhCvl8+kqhjWuT9vzleR6+7yNfLqNcqQxUNuNRxs44yx6Qz+eh6zpM10VRFKOgarMJIeGSof72LLRLrcBUkm5T4BECmXPf4xgZGYHruvB8H41Gg+WrK4qCPXv2sA73uq5jeXkZiwsL+PDDD6HIMiampjA1OYnx8fGWVLttu2RiE6GmqmgQRU0lQzpfSKxmz/fh+z50w4g0t0nnn/hDTQlWEEVIPB+pEpLjt1ji5G/6UyJkGsaseuqT9TwPOU2LmjqHYTSZAHDIZ0JEKX8cx0EQhOhHFCGQpslZiND3fVy/fp01Q94OLNuG57pQVbWFxDzPw+raGpaWlrCyvAzdMDAxPo6JyUkcPnQosmRTxhqvLO1nIuaAdJdMF+OIWephiHw+n0rIIdJ7e5qkIpVpxwgCyiMj96Wt4KOKIbEnwHEcisUiquvrcDwPiqLAcRzojcbWvpAZM2V4ngcnCPB8H2lUlkyD7JfU6b4mJyehyjLW1tawe8+eVLdHPp9nGTYIQ1TrdawsLeHGzZv45TvvoFQsYnJyEqOjoyiXSpEO/AD87bKigDfNSNI3hdj9IGCE6pEmy8Bm1kQQBJBUNSJbQQAPcn23YTG3s+LCMISsKMinWItBEDA/uO/70XjJ2B3HYRM+TwheEsWWCsg4bt++jUq53LWnbDd4JLWRF0UEQYDFxUWsr69jZXUVGxsbqFQqmJicxOnTpzEyOpqNqMnktR0t+i1pvlznFpS+76Op6wgIqbeLcaSNPwRQazTACwIK+TyCIECJrISyCrl9HjAk9hRomgZdUWC5LmRBQKFQgO/7aNTrKBFfIwPHgc9guYs8D58I/qchXvHJMmj6BJ2cLNvG4tISds/MdG5UzHGolMuolMs4cvQofN/Hxvo6lldWcOvWLdTrdTiOg1KphFKphEqlglKphHLyWrRB8kxypC+q7TiQRBGu58H1PHiOAy9WECSKIhRZhkgsYZ40XFBVlQXFBlklmwRrsJFCxjxZEQDRxBAPUPpkdeH7PvwgiColXZcJcImiyGRkOQBXr17Fs88+29cYPd9HvVZDrVbDyvIy6o0GmroOURBQIpPF0aNHMTY21rsbgmRjbTc9dcuKj6xwUg9J3C8BKSRqS+opGkkhoqYatm2jXC4zuRCFTM47CUNiTwElxvX1dThBwMi9VquhUa+jWC636m1k8LmLkhQ19OiQPkclSgdBVpqqRsvoIIjIfdeuzNkAgiBgfGIC4xMT7DXHcVCv11Gt1bC+vo7bt26hVq9DUZRoUhgZQblUQqlc7rrkFUQxWgUZBnKqyqxxURShKQpkScpmXd3nrBjaXrBjcDvlNeqSiYNa9XQSc1wXCEMsLi5CzeVQTFjryXuABjjr9TpqhMhrtRp0w0CpWISWyyGnaTixZw9GRke3n9IXKxTK1BKwC1p0iDqQelPXN0m9nTHSRvgOYYhGowGe51EsFCBKEnKFAorF4o7IhIljSOxtoJImHKbrQuJ5iITcG80m9GZzy83SLXtEEASA4+D7fnsJ2Fi+/HZJS5QkBLqO8bExrK6uYmFpCTNTU33f4LIsMx0bYFP1UNd11Go1VGs13Lp9G7V6HQ6xmMrlMrPuaYqjQ/
```

Comment:
This tutorial is a general one, not a finance one, Options shouldn't be mentioned here.